authorization/storage_authorization.go

```diff
@@ -71,42 +142,12 @@ func (s *Store) CreateAuthorization(ctx context.Context, tx kv.Tx, a *influxdb.A
 		a.ID = id
 	}
 
+	// Token must be unique to create authorization.
```

Comment:
If `s.uniqueAuthToken` fails should the mutation of `a.HashedToken = hashedToken` be reverted/rolled back at all? https://github.com/influxdata/EAR/issues/5819#issuecomment-2686492432